### PR TITLE
add dark-mode-trasitions

### PR DESCRIPTION
**Pull request by HoloPollock**
I thought it looks weird the nyou switched so I added some transitions I think I got them all

### CODE REVIEW

**Comment by qjack001** [comment]
lol i was also in the process of implementing this

File: src/components/Film.vue [hunk 1]
```diff
@@ -601,30 +595,33 @@ a:focus {
...
 .dark #advanced input[type="radio"]:checked + label {
 	border-color: #fff;
+	transition: border-color ease-in-out 0.25s;
 }
 
 #advanced input[type="radio"]:checked + label svg {
 	fill: #fff;
 }
 
 #advanced input[type="radio"]:focus + label {
```
Comment (qjack001) [inline src/components/Film.vue:617]: I would remove these two transitions (lines 619 and 624). While it does make the transition from light to dark slightly cleaner, it's a really small improvement—and the downside is that the box-shadow is animated when focusing on an option now (a pet peeve of mine).

**Review by qjack001 — CHANGES_REQUESTED** [review]

File: src/components/Film.vue (hunk 1 above)
Comment (qjack001) [reply src/components/Film.vue:617]: now that I'm thinking about it, it doesn't even improve the dark mode transition, because by clicking the switch the radio button will lose focus.

File: src/components/Film.vue [hunk 2]
```diff
@@ -766,6 +765,7 @@ button {
 
 .dark button {
 	background: #526e89;
+	transition: background ease-in-out 0.25s;
```
Comment (qjack001) [inline src/components/Film.vue:766]: Why is this needed? The property is already set in the standard button spec

File: src/components/Film.vue (hunk 1 above)
Comment (HoloPollock) [reply src/components/Film.vue:617]: done

File: src/components/Film.vue [hunk 3]
```diff
@@ -752,6 +750,7 @@ button {
 	line-height: 2.8rem;
 	border: 0;
 	background: black;
+	transition: background ease-in-out 0.25s;
```
Comment (qjack001) [inline src/components/Film.vue:751]: Also I hate that the button transitions on hover/focus now. I really liked that the site felt "snappy" in this way. Not sure if there's a way to have both

Comment (HoloPollock) [reply src/components/Film.vue:751]: yah maybe not. I think we can just remove this its fine

Comment (HoloPollock) [reply src/components/Film.vue]: I just removed it

**Review by qjack001 — APPROVED** [review]
